cfy_manager/config.py

```diff
@@ -113,9 +122,26 @@ def dump_config(self):
                 )
             )
 
-    def load_config(self):
+    def load_config(self, config_files=None):
         self._load_defaults_config()
-        self._load_user_config()
+        for config_file in config_files or [CONFIG_FILE_NAME]:
```

Comment:
```suggestion
        if config_files is None:
            config_files = [CONFIG_FILE_NAME]
        for config_file in config_files:
```
the `for x in y or z` syntax just seems unnatural, y'know

Reply:
OK.  And in your opinion `if config_files is None` is better than `if not config_files`, right?  Would we like to have the possibility to disable loading user configs by passing `config_files=[]` to the function?